adoc/chapters/programming_interface.adoc

```diff
@@ -19558,6 +19570,9 @@ the [code]#std::size_t# type.
 The element type parameter, [code]#DataT#, must be a _numeric type_ as it is
 defined by {cpp} standard.
 
+If [code]#DataT# is a trivially copyable type, then [code]#marray<DataT,
+NumElements># is a trivially copyable type.
```

Comment:
Side question: Is this `[code]` block in the  `html` well highlighted?  Just curious if we can add a new line in `[code]`

Reply:
Yes, it works as expected:
![image](https://github.com/user-attachments/assets/fc5c8117-6be3-4148-a54d-7dd868f070a0)